packages/appstore/src/components/transaction-list/completed-transaction/completed-transaction.tsx

```diff
@@ -1,14 +1,15 @@
 import React, { useMemo } from 'react';
+import { observer, useStore } from '@deriv/stores';
```

Comment:
@rostislav-deriv please sort the packages alphabetically

Reply:
Aren't they sorted correctly in this file @mahdiyeh-deriv?

Reply:
import { AppLinkedWithWalletIcon, Text, WalletIcon } from '@deriv/components';
import { observer, useStore } from '@deriv/stores';
import { Localize } from '@deriv/translations';

Reply:
Ah, you mean sorting lines by package names as well, ok

Reply:
Done